src/lib.rs

```diff
@@ -196,9 +196,11 @@ pub fn run() -> Result<(), BoxError> {
     io.add_method("add", move |params: Params| {
         let w: Result<WiFi, Error> = params.parse();
         match w {
-            Ok(w) => match network::add(&w) {
-                Ok(_) => Ok(Value::String("success".to_string())),
-                Err(_) => Err(Error::from(NetworkError::Add { ssid: w.ssid })),
+            Ok(w) => {
+                match network::add(&w) {
+                    Ok(_) => Ok(Value::String("success".to_string())),
+                    Err(e) =>           Err(Error::from(e))
```

Comment:
Looks like some extra spaces snuck in here? Running `cargo fmt` or `cargo +nightly fmt` prior to committing should fix this.

Reply:
@mycognosist oops, when I switched back to using mac I forgot to resetup auto-clippy,
but its now setup, I put cargo fmt and cargo clippy into .git/hooks/pre-commit for all my rust repositories, 
so hopefully that does the trick